posthog/migrations/0389_remove_prompts.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.1.13 on 2024-02-09 11:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
maybe we just mark the model as deprecated to avoid messing about with the migration?

Reply:
eugh... I hate leaving dead code lying around for no reason though...

Reply:
`test_migrations_are_safe.py` is your friend then....

IIRC I did write tests the last time I added an override to it

You're welcome to ahab this whale 🙈